ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -146,8 +147,21 @@ impl<'a> TraceBuilder<'a> {
                 let instridx = self.local_map[lvo.instr_id()];
                 jit_ir::Operand::Local(instridx)
             }
-            aot_ir::Operand::Constant(_co) => {
-                todo!()
+            aot_ir::Operand::Constant(co) => {
+                let c = self.aot_mod.constant(co);
+                let cidx = match self.aot_mod.const_type(c) {
+                    aot_ir::Type::Integer(it) => match it.num_bits() {
+                        32 => {
```

Comment:
Are the unwraps safe?

Reply:
If they fail we've messed up the lowering.

Reply:
Let's document that they can't fail unless there's a problem. [IMHO we should always document whether `unwrap`s can/can't fail. The assumptions we have when writing code are quickly forgotten and our future selves will scratch our heads wondering.]

Reply:
FWIW I've gotten into this habit. Even if it seems blindingly obvious now, it may not later.

Reply:
I've also realised the "safe" isn't a great word for unwrap, because it sounds like "unsafe" in Rust. So I've started to say "the unwrap can't fail because ...".

Reply:
Fixed in [eff1867](https://github.com/ykjit/yk/pull/993/commits/eff1867a494f3014c629e015e1796c2dc2426e97)